src/ui/Button/Button.tsx

```diff
@@ -0,0 +1,78 @@
+import { styled } from '../../stitches.config';
+
+export const Button = styled('button', {
+  // Reset
+  all: 'unset',
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  userSelect: 'none',
+  '&::before': {
+    boxSizing: 'border-box',
+  },
+  '&::after': {
+    boxSizing: 'border-box',
+  },
+  '& img': {
+    paddingRight: '$sm',
+  },
+  paddingLeft: '$sm2',
+  paddingRight: '$sm2',
+  paddingTop: '$sm',
+  paddingBottom: '$sm',
+
+  display: 'flex',
+  justifyContent: 'center',
+  height: '$xl2',
+  fontSize: '$md',
+  fontWeight: '$normal',
+  borderRadius: '$xs',
+
+  color: 'White',
+  variants: {
+    color: {
+      red: {
```

Comment:
primary rather than red,like in the demo:
![image](https://user-images.githubusercontent.com/4206097/147866333-9c5592e9-6a73-4121-ab6d-5c5bc6781afd.png)